delegates.rb

```diff
@@ -200,7 +200,7 @@ def authorize(options = {})
     image_id = context['identifier']
     type = derivative_type(context['resulting_size'])
     
-    return true if ['b','f','t','r'].include?(type) # POC to test response times. DO NOT merge this change to production. KAK. 
+    # return true if ['b','f','t','r'].include?(type) # POC to test response times. DO NOT merge this change to production. KAK. 

```

Comment:
so this will be reintroduced when we can add the column to filestore and implement the check for unpublished content?

Reply:
yup!